Library/Homebrew/cmd/update-bash.sh

```diff
@@ -208,6 +208,12 @@ update-bash() {
     esac
   done
 
+  if ! curl -I --connect-timeout 2 https://github.com &> /dev/null
```

Comment:
Please don't use `ping`, which doesn't support proxy. And even more, ping may be completely useless in certain network. e.g. In my university, ping is blocked by firewall as a security policy.